Fix wyoming missing network-online.target

techyporcupine opened the pull request:
## Description of changes

Added `wants = [ "network-online.target" ];` to the wyoming protocol related services to correct an issue seemingly caused by https://github.com/NixOS/nixpkgs/pull/258680.

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

OPNA2608 committed:
lomiri.lomiri-thumbnailer: init at 3.0.2

nikstur committed:
nixos/etc: optionally mount etc as an overlay

blitz committed:
nixos/dbus: explicitly set homeMode for dbus

Otherwise /run/dbus is created with the wrong permissions.

For some reason our recent changes made /run/dbus be 0700 instead of
0755. This is actually the default of homeMode. So something worked by
accident before?

nikstur committed:
nixos/profiles/perlless: init

jfroche committed:
opentelemetry-cpp: init at 1.13.0

Build with support for:

- in memory
- otlp http
- otlp grpc
- zipkin
- prometheus
- elasticsearch

Co-authored-by: tobim <tobim+github@fastmail.fm>
Co-authored-by: Nicolas Benes <nbenes.gh@xandea.de>

nikstur committed:
move-mount-beneath: init at unstable-2023-11-26

nikstur committed:
nixos/test-instrumentation: use file to set root password

nikstur committed:
nixos/systemd-sysusers: init

nikstur committed:
nixos/profiles/perlless: add release-note

nikstur committed:
nixos/etc: add release-note about overlay

nikstur committed:
nixos/systemd-sysusers: add release-note

nikstur committed:
nixos/systemd-sysusers: add manual section

nikstur committed:
nixos/etc: add manual section

nikstur committed:
nixos/profiles/perlless: add manual section

blitz committed:
linux: remove unused features

blitz committed:
doc: remove misleading kernel.features explanation

At this point kernel.features is more of an implementation detail and
normal users should not come into contact with it.

SuperSandro2000 committed:
nixos/archisteamfarm: don't use asf abbreviation for more clarity

OPNA2608 committed:
lomiri.qqc2-suru-style: init at 0.20230206

nicolas-goudry committed:
gitkraken: devendor git

r-ryantm committed:
python310Packages.google-cloud-bigquery-storage: 2.23.0 -> 2.24.0

zendo committed:
songrec: 0.3.3 -> 0.4.1

r-ryantm committed:
ocamlPackages.containers: 3.12 -> 3.13.1

nikstur committed:
systemdUkify: init

nikstur committed:
nixos/uki: init

nikstur committed:
nixos/tests/appliance-repart-image: use UKIs

OPNA2608 committed:
ayatana-indicator-datetime: init at 23.10.0

OPNA2608 committed:
ayatana-indicator-datetime: 23.10.0 -> 23.10.1

greizgh committed:
f3d: mark broken

As of 2024-01-20, running f3d result in:
error while loading shared libraries: libvtkInteractionWidgets.so.1: cannot open shared object file: No such file or directory

Related issue: https://github.com/NixOS/nixpkgs/issues/262328

r-ryantm committed:
python310Packages.spglib: 2.1.0 -> 2.2.0

Stunkymonkey committed:
python311Packages.requests-download: rename from requests_download

synecdokey committed:
nodePackages: update to latest

orivej committed:
davinci-resolve: add street to fix build

Fixes build after #274007

Without the street the build fails with:

downloadid is f38fc270c4c040bea4782182d0fbb367
resolveurl is Error: Must register to be able to perform the download
curl: (3) URL rejected: Malformed input to a URL function

buckley310 committed:
vmware-horizon-client: 2306 -> 2309.1

vbgl committed:
framac: fix build with OCaml ≥ 5

vbgl committed:
advi: use OCaml 4.14

vbgl committed:
coccinelle: use OCaml 4.14

vbgl committed:
comby: use OCaml 4.14

vbgl committed:
coqPackages.compcert: use OCaml 4.14

vbgl committed:
cubicle: use OCaml 4.14

vbgl committed:
flow: use OCaml 4.14

vbgl committed:
fstar: use OCaml 4.14

vbgl committed:
guestfs-tools: use OCaml 4.14

vbgl committed:
heptagon: use OCaml 4.14

vbgl committed:
jackline: use OCaml 4.14

vbgl committed:
libbap: use OCaml 4.14

vbgl committed:
libguestfs: use OCaml 4.14

vbgl committed:
liquidsoap: use OCaml 4.14

vbgl committed:
megam: use OCaml 4.14

vbgl committed:
orpie: use OCaml 4.14

vbgl committed:
python3Packages.bap: use OCaml 4.14

vbgl committed:
rml: use OCaml 4.14

vbgl committed:
satallax: use OCaml 4.14

vbgl committed:
satysfi: use OCaml 4.14

vbgl committed:
spark2014: use OCaml 4.14

vbgl committed:
supermin: use OCaml 4.14

vbgl committed:
teyjus: use OCaml 4.14

vbgl committed:
virt-top: use OCaml 4.14

vbgl committed:
ocaml: default to version 5.1

RaitoBezarius committed:
nixos/networking/keepalived: do not emit `unicastPeers` when there are none

In the past, we emitted `unicast_peer` block even with empty unicast peers list.

This now results in:
```
Keepalived_vrrp[392762]: A unicast keyword has been specified without any unicast peers. Defaulting to multicast. This usage is deprecated - please update your configuration.
```

We do not emit it anymore to avoid this warning.

samueltardieu committed:
remove: rtthost

Since release 0.22, `rtthost` is built as part of `probe-rs`.

simonhollingshead committed:
nixos/auto-upgrade: add system.autoUpgrade.fixedRandomDelay

From systemd 247, timers can be configured to randomize their delay
once and to apply the same random delay for all future executions.
This allows users to have less jitter between auto-upgrade executions
while still avoiding multiple timers firing simultaneously on a
machine or multiple machines all performing their upgrades
simultaneously.

The default option value (false) is backwards compatible.

All supported versions of NixOS (in fact, back to and including 21.05)
use systemd 247 or later.

kirillrdy committed:
go-musicfox: unpin go

kirillrdy committed:
k0sctl: unpin go

kirillrdy committed:
kubectl-klock: unpin go

kirillrdy committed:
timoni: unpin go

kirillrdy committed:
ktailctl: unpin go

kirillrdy committed:
static-server: unpin go

kirillrdy committed:
uplosi: unpin go

kirillrdy committed:
zitadel: unpin go

kirillrdy committed:
risor: unpin go

kirillrdy committed:
turso-cli: unpin go

kirillrdy committed:
juicity: unpin go

kirillrdy committed:
netdata-go-plugins: unpin go

Mic92 committed:
nixos/buildbot: don't require network-online.target

The daemon setup itself only requires the firewall to be up.
Workers will connect itself as they can reach the master instance.
That's why we don't need an online target for buildbot to function.

alyssais committed:
mailmanPackages.hyperkitty: 1.3.7 -> 1.3.8

alyssais committed:
mailman: 3.3.8 -> 3.3.9

alyssais committed:
mailmanPackages.postorius: 1.3.8 -> 1.3.10

alyssais committed:
mailman-web: 0.0.6 -> 0.0.8

alyssais committed:
mailmanPackages.python3.pkgs.redis: 4.5.4 -> 4.6.0

alyssais committed:
mailmanPackages.hyperkitty: remove obsolete patches

The test failure only happened with Python 3.10.  We use 3.11 by
default, and there's not really any reason to change it, so I don't
think we need to keep that patch around.

alyssais committed:
mailman: remove obsolete substitution

umlx5h committed:
gtrash: init at 0.0.5

umlx5h committed:
maintainers: add umlx5h

rhoriguchi committed:
gphotos-sync: 3.1.2 -> 3.2.1

r-ryantm committed:
pdfhummus: 4.6.2 -> 4.6.3

jmbaur committed:
xilinx-bootgen: xilinx_v2023.1 -> xilinx_v2023.2

https://github.com/xilinx/bootgen/compare/xilinx_v2023.1...xilinx_v2023.2

mfrischknecht committed:
foxotron: fetch assimp patches to fix build

Foxotron vendors a lot of dependencies (see the `external` directory and
the various submodules it contains). One of them (`FBX`) fails to build
with current versions of GCC because the compiler cannot resolve the
type `std::uint32_t`. The affected file still imports the type via
`#include <stdint.h>`, which has been superseded by `#include <cstdint>`
in C++11.

The upstream patch applied here fixes this build issue by introducing
the missing include line of `<cstdint>`.

The new addition to `postPatch` disables a `-Werror` setting that
(currently) seemingly only causes assimp builds failures on darwin.

Co-authored-by: Cosima Neidahl <opna2608@protonmail.com>

r-ryantm committed:
pocketbase: 0.20.5 -> 0.20.7

r-ryantm committed:
apt: 2.7.8 -> 2.7.9

LostAttractor committed:
qq: 3.2.3-20201 -> 3.2.5-20979

sikmir committed:
josm: 18907 → 18940

SomeoneSerge committed:
cudaPackagesGoogle.cudnn_8_6: fix eval

Now:
❯ nix eval -f . --arg config '{ allowUnfree = true; }' cudaPackagesGoogle.cudnn_8_6.outPath
"/nix/store/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-cudnn-8.6.0.163"

Previously:
meta.broken = true

SomeoneSerge committed:
cudaPackagesGoogle.cudnn_8_6: ensure present on all platforms

...including jetson

Previously:
❯ nix eval -f . --argstr system aarch64-linux --arg config '{ cudaCapabilities = [ "6.2" ]; cudaEnableForwardCompat = false; cudaSupport = true; allowUnfree = true; }' -L cudaPackagesGoogle.cudnn_8_6.outPath
attribute ... in selection path .... not found

Now:
❯ nix eval -f . --argstr system aarch64-linux --arg config '{ cudaCapabilities = [ "6.2" ]; cudaEnableForwardCompat = false; cudaSupport = true; allowUnfree = true; }' -L cudaPackagesGoogle.cudnn_8_6.outPath
       error: Package ‘cudnn-8.6.0.163’ in ... is not available on the requested hostPlatform:

dasJ committed:
nixos/nix-gc: Drop with lib;

dasJ committed:
nixos/nix-gc: Add Type so systemctl waits properly

dasJ committed:
nixos/nix-gc: Use singleLineStr where possible

r-ryantm committed:
zpaqfranz: 58.11 -> 59.1

r-ryantm committed:
cemu: 2.0-61 -> 2.0-65

OPNA2608 committed:
lomiri.qqc2-suru-style: Add meta.changelog

OPNA2608 committed:
lomiri.qtmir: init at 0.7.2-unstable-2024-01-08

wegank committed:
ocamlPackages.containers: set minimal ocaml version

OPNA2608 committed:
lomiri.telephony-service: init at 0.5.2

zowoq committed:
hydra_unstable: drop patch

`allowed-uris` fix is included in nix 2.19.3
https://github.com/NixOS/nix/commit/b38e5a665e9d0aa7975beb0ed12e42d13a392e74

teto committed:
backblaze2-b2: support alternative executable name

according to official doc backblaze.com/docs/cloud-storage-command-line-tools mentions the executable
is supposed to be called `b2`. Several distributions including nixpkgs renamed it to `backblaze-b2`
but it is annoying if you dont use boost's b2.
This commit makes it possible to override the executable name to
something else.

SuperSandro2000 committed:
nixos/archisteamfarm: drop with lib

aaronjheng committed:
spirit: unstable-2023-12-15 -> 0-unstable-2024-01-11

r-ryantm committed:
doitlive: 4.3.0 -> 5.0.0

marsam committed:
resvg: 0.37.0 -> 0.38.0

Diff: https://github.com/RazrFalcon/resvg/compare/v0.37.0...v0.38.0

Changelog: https://github.com/RazrFalcon/resvg/blob/v0.38.0/CHANGELOG.md

marsam committed:
luau: 0.607 -> 0.609

Diff: https://github.com/luau-lang/luau/compare/0.607...0.609

Changelog: https://github.com/luau-lang/luau/releases/tag/0.609

marsam committed:
gallery-dl: 1.26.6 -> 1.26.7

Changelog: https://github.com/mikf/gallery-dl/blob/v1.26.7/CHANGELOG.md

marsam committed:
luau: add meta.mainProgram

r-ryantm committed:
nanovna-saver: 0.6.0 -> 0.6.3

r-ryantm committed:
patatt: 0.6.2 -> 0.6.3

trofi committed:
diffoscope: 253 -> 254

Changes: https://diffoscope.org/news/diffoscope-254-released/

r-ryantm committed:
fluent-bit: 2.2.1 -> 2.2.2

r-ryantm committed:
python312Packages.langsmith: 0.0.80 -> 0.0.83

r-ryantm committed:
python311Packages.crytic-compile: 0.3.5 -> 0.3.6

r-ryantm committed:
vscodium: 1.85.1.23348 -> 1.85.2.24019

r-ryantm committed:
php81Extensions.phalcon: 5.5.0 -> 5.6.0

mfrw committed:
tailspin: 2.4.0 -> 3.0.0

Diff: https://github.com/bensadeh/tailspin/compare/2.4.0...3.0.0
Changelog: https://github.com/bensadeh/tailspin/blob/3.0.0/CHANGELOG.md
Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>

GaetanLepage committed:
obs-studio-plugins.obs-composite-blur: init at 1.1.0

chvp committed:
qtile: 0.23.0 -> 0.24.0

https://github.com/qtile/qtile/releases/tag/v0.24.0 (changelog: https://github.com/qtile/qtile/blob/master/CHANGELOG)

r-ryantm committed:
vcpkg: 2023.12.12 -> 2024.01.12

r-ryantm committed:
unifi-protect-backup: 0.10.2 -> 0.10.3

wegank committed:
stormlib: migrate to by-name, rename from StormLib

wegank committed:
stormlib: fix build on darwin

r-ryantm committed:
oxker: 0.5.0 -> 0.6.0

r-ryantm committed:
kubecm: 0.26.0 -> 0.27.1

wegank committed:
_9base: fix build on x86_64-darwin

fabaff committed:
python311Packages.msal: 1.25.0 -> 1.26.0

Changelog: https://github.com/AzureAD/microsoft-authentication-library-for-python/releases/tag/1.26.0

fabaff committed:
python311Packages.msal: refactor

fabaff committed:
python311Packages.msal-extensions: 1.0.0 -> 1.1.0

Diff: https://github.com/AzureAD/microsoft-authentication-extensions-for-python/compare/refs/tags/1.0.0...1.1.0

Changelog: https://github.com/AzureAD/microsoft-authentication-extensions-for-python/releases/tag/1.1.0

fabaff committed:
python311Packages.python-gitlab: 4.2.0 -> 4.4.0

Changelog: https://github.com/python-gitlab/python-gitlab/blob/v4.4.0/CHANGELOG.md

piegamesde committed:
Revert "CODEOWNERS: Watch the maintainers lists"

fabaff committed:
python311Packages.pybids: relax formulaic

fabaff committed:
python311Packages.qingping-ble: 0.9.0 -> 0.10.0

Diff: https://github.com/bluetooth-devices/qingping-ble/compare/refs/tags/v0.9.0...v0.10.0

Changelog: https://github.com/Bluetooth-Devices/qingping-ble/blob/v0.10.0/CHANGELOG.md

r-ryantm committed:
kaniko: 1.19.2 -> 1.20.0

wegank committed:
_9base: move flags to env

fabaff committed:
python311Packages.parsedmarc: refactor

r-ryantm committed:
go-containerregistry: 0.17.0 -> 0.18.0

r-ryantm committed:
gdbgui: 0.15.1.0 -> 0.15.2.0

wegank committed:
sony-headphones-client: fix build with gcc 13

r-ryantm committed:
python311Packages.weaviate-client: 3.26.0 -> 3.26.1

fabaff committed:
nvdtools: init at 0.1.5

Tools to work with the feeds (vulnerabilities, CPE dictionary etc.) distributed by National Vulnerability Databas

https://github.com/facebookincubator/nvdtools

fabaff committed:
python311Packages.yaramod: 3.20.2 -> 3.21.0

Diff: https://github.com/avast/yaramod/compare/refs/tags/v3.20.2...v3.21.0

Changelog: https://github.com/avast/yaramod/blob/v3.21.0/CHANGELOG.md

fabaff committed:
python311Packages.yfinance: 0.2.35 -> 0.2.36

Diff: https://github.com/ranaroussi/yfinance/compare/refs/tags/0.2.35...0.2.36

Changelog: https://github.com/ranaroussi/yfinance/blob/0.2.36/CHANGELOG.rst

fabaff committed:
python311packages.types-aiobotocore-*: 2.9.0 -> 2.11.0

fabaff committed:
python311Packages.vulture: 2.10 -> 2.11

Changelog: https://github.com/jendrikseipp/vulture/releases/tag/v2.11

mweinelt committed:
home-assistant: 2024.1.3 -> 2024.1.5

https://github.com/home-assistant/core/releases/tag/2024.1.4
https://github.com/home-assistant/core/releases/tag/2024.1.5

r-ryantm committed:
usbsdmux: 0.2.1 -> 24.1

wegank committed:
python312Packages.python-trovo: fix src

r-ryantm committed:
edir: 2.22 -> 2.27

r-ryantm committed:
ipatool: 2.1.3 -> 2.1.4

r-ryantm committed:
lefthook: 1.5.6 -> 1.5.7

r-ryantm committed:
netbird: 0.25.2 -> 0.25.4

r-ryantm committed:
gato: 1.5 -> 1.6

khaneliman committed:
fastfetch: 2.6.1 -> 2.6.3

r-ryantm committed:
python311Packages.python-frontmatter: 1.0.1 -> 1.1.0

r-ryantm committed:
trustymail: 0.8.1 -> 0.8.3

r-ryantm committed:
rspamd: 3.7.5 -> 3.8.0

r-ryantm committed:
oh-my-posh: 19.6.0 -> 19.8.0

r-ryantm committed:
reshape: 0.6.1 -> 0.7.0

r-ryantm committed:
sslscan: 2.1.2 -> 2.1.3

r-ryantm committed:
mkp224o: 1.6.1 -> 1.7.0

r-ryantm committed:
abracadabra: 2.3.4 -> 2.3.5

r-ryantm committed:
cinnamon.mint-artwork: 1.7.9 -> 1.8.0

Updates etc/skel/.config/qt5ct/qt5ct.conf, which does not affect us.

r-ryantm committed:
hblock: 3.4.2 -> 3.4.3

JerrySM64 committed:
linuxKernel.kernels.linux_zen: 6.7-zen3 -> 6.7.1-zen1

lf- commented:
This needs a rebase to only one commit and follow CONTRIBUTING.md, something like: `nixos/wyoming: fix missing network-online.target`.

r-ryantm committed:
audible-cli: 0.2.5 -> 0.2.6

r-ryantm committed:
python311Packages.gvm-tools: 23.11.0 -> 24.1.0

r-ryantm committed:
pt2-clone: 1.65.1 -> 1.66.1

r-ryantm committed:
python311Packages.diffusers: 0.25.0 -> 0.25.1

r-ryantm committed:
python311Packages.qutip: 4.7.3 -> 4.7.4

K900 committed:
yuzuPackages.compat-list: unstable-2024-01-08 -> unstable-2024-01-21

r-ryantm committed:
python311Packages.python-songpal: 0.16 -> 0.16.1

K900 committed:
yuzuPackages.mainline: 1676 -> 1689

Changelog: https://yuzu-emu.org/entry

fabaff committed:
python311Packages.monty: refactor

K900 committed:
yuzu-ea: 4056 -> 4079

datafoo committed:
vscode-extensions.elixir-lsp.vscode-elixir-ls: 0.18.1 -> 0.19.0

https://github.com/elixir-lsp/vscode-elixir-ls/blob/master/CHANGELOG.md#v0190-21-january-2023

r-ryantm committed:
monolith: 2.8.0 -> 2.8.1

r-ryantm committed:
tailscale-nginx-auth: 1.56.1 -> 1.58.0

fabaff committed:
python311Packages.monai-deploy: refactor

r-ryantm committed:
syft: 0.100.0 -> 0.101.1

K900 committed:
kdsoap: fix after update

fabaff committed:
python311Packages.githubkit: add optional-dependencies

r-ryantm committed:
gh-dash: 3.11.1 -> 3.12.0

fabaff committed:
python311Packages.anysqlite: init at 0.0.5

Sqlite3 for asyncio and trio

https://github.com/karpetrosyan/anysqlite

fabaff committed:
python311Packages.hishel: init at 0.0.21

HTTP Cache implementation for HTTPX and HTTP Core

https://github.com/karpetrosyan/hishel

r-ryantm committed:
fuc: 1.1.10 -> 2.0.0

jnsgruk committed:
homepage-dashboard: 0.8.4 -> 0.8.6

https://github.com/gethomepage/homepage/releases/tag/v0.8.6

r-ryantm committed:
b4: 0.12.3 -> 0.12.4

Munksgaard committed:
elixir-ls: 0.18.1 -> 0.19.0

fabaff committed:
python311Packages.githubkit: 0.10.7 -> 0.11.0

Diff: yanyongyu/githubkit@refs/tags/v0.10.7...v0.11.0

Changelog: https://github.com/yanyongyu/githubkit/releases/tag/v0.11.0

r-ryantm committed:
goflow2: 2.1.1 -> 2.1.2 (#282835)

Sohalt committed:
babashka: 1.3.186 -> 1.3.188

r-ryantm committed:
percollate: 4.0.4 -> 4.0.5

r-ryantm committed:
zsh-prezto: unstable-2023-11-30 -> unstable-2024-01-18

r-ryantm committed:
xe-guest-utilities: 8.3.1 -> 8.4.0

trofi committed:
postfix: 3.8.4 -> 3.8.5

Changes: https://www.postfix.org/announcements/postfix-3.8.5.html

r-ryantm committed:
tuba: 0.6.1 -> 0.6.2

r-ryantm committed:
pg_activity: 3.0.3 -> 3.4.2

teutat3s committed:
node-manta: 5.4.1 -> 5.4.2

https://github.com/TritonDataCenter/node-manta/blob/v5.4.2/CHANGES.md#542

r-ryantm committed:
jackett: 0.21.1510 -> 0.21.1588

r-ryantm committed:
seaweedfs: 3.61 -> 3.62

Sohalt committed:
build-graalvm-native-image: do not use NATIVE_IMAGE_DEPRECATED_BUILDER_SANITATION

Packages that require access to environment variables in the build
should specify these using `-E` arguments in `nativeImageBuildArgs` or
using a `native-image.properties` as described here:
https://www.graalvm.org/22.1/reference-manual/native-image/BuildConfiguration/#embedding-a-configuration-file

Specifying NATIVE_IMAGE_DEPRECATED_BUILDER_SANITATION=true breaks with
packages that list their environment variables explicitly in `native-image.properties`.

r-ryantm committed:
balena-cli: 17.4.11 -> 17.4.12

alyssais committed:
linux/hardened/patches/6.1: 6.1.73-hardened1 -> 6.1.74-hardened1

alyssais committed:
linux/hardened/patches/6.6: 6.6.12-hardened1 -> 6.6.13-hardened1

alyssais committed:
linux_testing: 6.7 -> 6.8-rc1

r-ryantm committed:
nats-server: 2.10.7 -> 2.10.9

r-ryantm committed:
sd-local: 1.0.51 -> 1.0.52

fabaff committed:
python312Packages.ipwhois: add patch to fix assertEquals usage

sequencer committed:
circt: 1.62.0 -> 1.63.0

wathiede committed:
fstl: 0.9.4 -> 0.10.0 (#264257)

* fstl: 0.9.4 -> 0.10.0

Note that https://github.com/mkeeter/fstl now redirects to
https://github.com/fstl-app/fstl. It would appear the old repo is no
longer updated, and the version under fstl-app is the proper location.

* fstl: address pull request feedback

* fstl: change install hook usage per review suggestion.

yayayayaka committed:
element-{web,desktop}: 1.11.54 -> 1.11.55

https://github.com/element-hq/element-desktop/releases/tag/v1.11.55
https://github.com/element-hq/element-web/releases/tag/v1.11.55

picnoir committed:
ncncd: unstable-2023-10-26 -> unstable-2024-01-16

All our modifications have been merged upstream. There's no need to
point to the Nix Community fork anymore, pointing to
upstream (twosigma) instead.

Fixes #282485

sheepforce committed:
openmolcas: fix pyparsing >= 3.11 compatibility

r-ryantm committed:
vals: 0.32.0 -> 0.33.0

r-ryantm committed:
dcnnt: 0.9.0 -> 0.9.2

r-ryantm committed:
eigenmath: unstable-2023-12-31 -> unstable-2024-01-22

mweinelt committed:
python311Packages.homeassistant-stubs: 2024.1.3 -> 2024.1.5

https://github.com/KapJI/homeassistant-stubs/releases/tag/2024.1.4
https://github.com/KapJI/homeassistant-stubs/releases/tag/2024.1.5

r-ryantm committed:
python311Packages.snakemake-interface-common: 1.15.0 -> 1.15.1

minijackson committed:
netbox: 3.6.9 -> 3.7.1

Or another way to see it:

netbox_3_7: init at 3.7.1

Make NetBox 3.7 the default version if stateVersion >= 24.05,
switch upgrade test to test upgrade from 3.6 to 3.7,
remove clearcache command for >=3.7.0,
make reindex command mandatory

minijackson committed:
netbox_3_5: remove

minijackson committed:
netbox_3_6: mark as EOL

minijackson committed:
nixos/rl-2405: document NetBox breaking change

yayayayaka committed:
matrix-sliding-sync: 0.99.14 -> 0.99.15

https://github.com/matrix-org/sliding-sync/releases/tag/v0.99.15

r-ryantm committed:
yazi: 0.2.1 -> 0.2.2

r-ryantm committed:
er-patcher: 1.10-1 -> 1.10.1-1

r-ryantm committed:
cirrus-cli: 0.108.2 -> 0.108.5

Sohalt committed:
build-graalvm-native-image: pass whole environment

Pass the whole environment to the native-image build process by
generating a -E option for every environment variable.

This has the same effect as setting
NATIVE_IMAGE_DEPRECATED_BUILDER_SANITATION=true
but is compatible with packages providing -E options themselves

r-ryantm committed:
tuifimanager: 3.3.1 -> 3.3.5

r-ryantm committed:
dracula-theme: unstable-2024-01-08 -> unstable-2024-01-16

r-ryantm committed:
pypi-mirror: 5.0.2 -> 5.2.0

TomaSajt committed:
simulide_1_1_0: use corrected upstream source

dotlambda committed:
Revert "Dovecot: Do not include empty sieve_extensions and sieve_global_extensions"

r-ryantm committed:
boxed-cpp: 1.1.0 -> 1.2.0

r-ryantm committed:
innernet: 1.6.0 -> 1.6.1

r-ryantm committed:
shopware-cli: 0.4.5 -> 0.4.13

r-ryantm committed:
python311Packages.willow: 1.7.0 -> 1.8.0

r-ryantm committed:
protolint: 0.47.4 -> 0.47.5

r-ryantm committed:
esphome: 2023.12.5 -> 2023.12.8

AndersonTorres committed:
melonDS: 0.9.5 -> 0.9.5-unstable-2024-01-17 (#282620)

* melonDS: refactor

- extra dependency: zstd
- finalAttrs design pattern
- get rid of nested with
- add AndersonTorres as maintainer

* melonDS: set strictDeps as true

- Why does it not find extra-cmake-modules in nativeBuildInputs?

  Let's mark it on issue https://github.com/NixOS/nixpkgs/issues/178468

* melonDS: 0.9.5 -> 0.9.5-unstable-2024-01-17

r-ryantm committed:
prometheus-pushgateway: 1.6.2 -> 1.7.0 (#282708)

r-ryantm committed:
norminette: 3.3.54 -> 3.3.55

r-ryantm committed:
cargo-zigbuild: 0.18.1 -> 0.18.2

r-ryantm committed:
minio-client: 2024-01-05T05-04-32Z -> 2024-01-18T07-03-39Z

r-ryantm committed:
python311Packages.cantools: 39.4.2 -> 39.4.3

RaitoBezarius committed:
keepalived: add myself as a maintainer

I will use this for a while, so… let's keep it tidy.

Stunkymonkey committed:
python311Packages.readme-renderer: rename from readme_renderer (#279385)

techyporcupine commented:
Would it be fine for me to completely close and recreate this PR? I did a couple things wrong when making it (specifically doing it all from GitHub web)

lf- commented:
You can fix it with the command line:

```
git remote add mine https://github.com/techyporcupine/nixpkgs
git fetch mine
git switch fix-wyoming-missing-network-online.target
(rebase on top of the parent of the first commit)
git rebase -i e1aa7a3~
```

Delete the line with the merge commit, change the first one to r for reword, change the next two to f for fixup. Quit your editor. Change the commit message when it gives you an editor again.

You can then open the whole thing in `gitk` or your preferred git viewer and take a look, check it's as expected.

Then you can `git push -f mine HEAD:fix-wyoming-missing-network-online.target`

Note: I came up with this in my head, this is how I would fix it, cannot guarantee it works word for word.

drupol commented:
I'm locking and closing this pull request, because through the rebase you inadvertently requested many maintainers for review, which subscribed them to update notifications, resulting in unnecessary spam inside everyone's inbox. While we can remove their review requests, we sadly cannot unsubscribe anyone.

Please create a new pull request and for the next time remember to set your PR to draft status before rebasing. In draft status, you can preview the list of maintainers that are about to be requested for review, which allows you to sidestep this issue.
